### ISSUE TITLE
Add support for lineage schema to the python and R libraries

### ISSUE THREAD

**Issue by javh** [airr-community/airr-standards#378]
Once the schemas are closer to finalized, we'll need to add support for the Clone, Tree and Node schemas to the python and R libraries.

**Comment by javh** [comment]
Schema should be finalized enough to implement R/python library support.

**Comment by eharkins** [comment]
Here is the example data from Olmsted:  https://github.com/matsengrp/olmsted/blob/master/example_data/full_schema_dataset.json. 

It conforms to the [Olmsted schema](http://www.olmstedviz.org/schema.html), which extends the AIRR schema, but hasn't been updated recently if the AIRR schema has had major changes.

**Comment by bcorrie** [comment]
@javh same for this one - has this been done and if not do we want to support it for v2.0?

The way I would think of this is that if an schema object (e.g. Tree/Node) is moving from "Experimental" to a full schema object in v2.0, then I would expect the AIRR libraries to support objects of that type.  

My question would be are these robust enough to be not experimental? I am not sure they have been used by anyone in the community. So perhaps not?

**Comment by scharch** [comment]
This should be assigned to me - I thought it already was.
Let's put the lineage schema on the agenda for next meeting.  Back in October-ish we discussed if we should try simplifying to the bits needed for the most common use cases. The big complex current schema was primarily developed with an eye toward Olmsted, which was never formally published and Erick says isn't being actively developed.

**Comment by bcorrie** [comment]
Related #333

**Comment by javh** [comment]
> has this been done and if not do we want to support it for v2.0?

Yeah, object support is sufficiently abstracted now that these should work. However, I don't see unit test data that includes `Clone` (where this would start), so I think we'll need that to be sure.

**Comment by bussec** [comment]
@scharch I put it on the [agenda for the next call](https://github.com/airr-community/airr-standards/wiki/Minutes_Standards_2024-03).

**Comment by javh** [comment]
Once #778 is merged in we can use the example from there for these tests.
